layouts/partials/logo.html

```diff
@@ -0,0 +1 @@
+<img src="/img/logo.svg" alt="">
```

Comment:
```suggestion
{{if eq .Site.Language.Lang "en"}}
<a href="/en">
  <img src="/img/logo_en.svg" alt="" />
</a>
{{ else if eq .Site.Language.Lang "fr"}}
<a href="/fr">
  <img src="/img/logo.svg" alt="" />
</a>
{{end}}
```

J'ajouterais un if comme ça pour montrer le logo Open Science UMontreal (EN) vs Open Science UMontréal (FR) selon la langue choisie.
J'ai aussi ajouté le lien comme ça si on clique dessus, ça nous ramène à Home.